akka-docs/src/test/scala/docs/dispatcher/MyUnboundedMailbox.scala

```diff
@@ -15,9 +15,6 @@ import com.typesafe.config.Config
 import java.util.concurrent.ConcurrentLinkedQueue
 import scala.Option
 
-// Marker trait used for mailbox requirements mapping
-trait MyUnboundedMessageQueueSemantics
-
```

Comment:
Refer to [this comment](https://github.com/akka/akka/pull/23191/files#r122614439).